src/Markup/Avalonia.Markup/Markup/Data/DelayedBinding.cs

```diff
@@ -38,7 +38,7 @@ public static void Add(IStyledElement target, AvaloniaProperty property, IBindin
             else
             {
 
-                if (!_entries.TryGetValue(target, out List<Entry> bindings))
+                if (!_entries.TryGetValue(target, out var bindings))
```

Comment:
`Avalonia.Markup` has nullable warnings as errors enabled, and `net6.0` is better at finding nullable problems.